src/main/java/org/galatea/starter/service/IexClient.java

```diff
@@ -20,7 +20,7 @@ public interface IexClient {
    *
    * @return a list of all of the stock symbols supported by IEX.
    */
-  @GetMapping("/ref-data/symbols")
+  @GetMapping("/ref-data/symbols?token=pk_73824a25287d45ccb3d27bd8003c707c")
```

Comment:
I think you're on the right path but this isn't quite correct.  I'm not sure what this token value is (I'm assuming some type of authentication?) but the goal of GetMapping is to help direct a rest call to a service in your code.